### PR TITLE
chore: Add ESLint packages

### PR DESCRIPTION
**Pull request by dkimura**
## TL;DR

- Add necessary ESLint packages for `eslint-config-globis`

## Check list

- [ ] Passed CI
- [ ] Required packages installed?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/globis-org/frontend-standard/issues"
+  "scripts": {
+    "lint": "yarn workspaces run test",
+    "test": "yarn workspaces run test"
   },
-  "homepage": "https://github.com/globis-org/frontend-standard#readme",
   "devDependencies": {
+    "eslint": "6.7.2",
```
Comment (dkimura) [inline package.json:25]: 忘れてた。。

Comment (shoota) [reply package.json:25]: oh

**Review by shoota — APPROVED** [review]
LGTM

File: packages/eslint-config-globis/package.json [hunk 2]
```diff
@@ -20,5 +20,20 @@
   "scripts": {
     "test": "echo test",
     "lint": "echo lint"
+  },
+  "peerDependencies": {
+    "eslint": "^6.7.2",
+    "prettier": "^1.19.1"
```
Comment (tatsushitoji) [inline packages/eslint-config-globis/package.json]: These do not fix the version ?

Comment (dkimura) [reply packages/eslint-config-globis/package.json]: It is not fixed.
because it is free for the user.

Comment (tatsushitoji) [reply packages/eslint-config-globis/package.json]: OK that the user can select version.
However, is disable PeerDependencies setting necessary for renovate app ?

Comment (dkimura) [reply packages/eslint-config-globis/package.json]: aha. done~.
https://github.com/globis-org/frontend-standard/pull/3/commits/4ef67734a6a2da77e19d57605c16a0dcd8375ece

**Review by tatsushitoji — APPROVED** [review]
LGTM!

**Review by okmttdhr — COMMENTED** [review]
でおくれた

**Pull request merged**